patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+InnerSource Hackathon
```

Comment:
Just brainstorming:
If you could not use "InnerSource" in the title, how might you call this pattern?

Reply:
I am not sure. The closest I can think off is - 'Hackathon: Safe experiment space', but I feel it doesn't sound right. I took inspiration from other patterns - InnerSource Portal and InnerSource License, which give the user a good understanding of what to expect in the pattern, just by the title. 

Please feel free to suggest one :)

Reply:
Maybe it would help to include something that the Hackathon achieves in the title?

Something that expresses the gained practical experience?

Though I can follow the argument of @spriya-m referring to InnerSource Portal and -License as examples.